### DISCUSSION TITLE
Github Action for MinVer

### DISCUSSION THREAD

**Discussion by tobias-graf-p** [adamralph/minver#850] [General]
Hi

I was wondering whether you (@adamralph) as the maintainer of MinVer plan to provide an "official" Github action to use MinVer as a standalone workflow step. The use case for this requirement is described here: https://github.com/thefringeninja/action-minver/issues/20#issuecomment-1354309051 and here: https://github.com/adamralph/minver/discussions/793 (or here respectively: https://stackoverflow.com/a/73472371/7523752).

The maintainer of the first MinVer Github action seems to not continue to maintain it ... See here: https://github.com/thefringeninja/action-minver/issues/20

There is a fork of that initial action which is more feature-rich and more up-to-date. There the plan of the maintainer is to create a dotnet Github action, see here: https://github.com/Stelzi79/action-minver/issues/17#issuecomment-1369001499

Wouldn't it be a great addition to MinVer to provide an "official" Github action from the maintainer of the tool itself?

**Comment by adamralph** [comment]
My response to this is similar to https://github.com/adamralph/minver/issues/1006#issuecomment-2241224677 — it's not something I need, so this won't be provided.
